fix(core): stricter event emitter type (#38916)
angular/angular: aio/content/start/index.md

  <code-example path="getting-started/src/app/product-alerts/product-alerts.component.ts" header="src/app/product-alerts/product-alerts.component.ts" region="input-output"></code-example>

>   Note how the `void` type is used to specify that `notify.emit()` (below) won't accept any argument.

Reviewer: ```suggestion
You use the `void` type to specify that `notify.emit()` doesn't accept an argument.
```

Just switches to the active from the passive voice.
